console/style.rst

```diff
@@ -67,8 +67,8 @@ title of the command::
 
         protected function execute(InputInterface $input, OutputInterface $output)
         {
-            $io = new SymfonyStyle($input, $output);
-            $io->title('Lorem Ipsum Dolor Sit Amet');
+            $inputOutput = new SymfonyStyle($input, $output);
```

Comment:
Do not really like this one but it is better than `$io`, maybe `$symfonyStyle` if that makes sense?

Reply:
I think it's better to keep `$io` because we use it everywhere in the docs, the code, the Symfony Demo app, etc.